lldb/test/Shell/Disassemble/TestDisassembleRISCVXqciInstrutions.test

```diff
@@ -0,0 +1,10 @@
+# REQUIRES: target=riscv32-{{.*}}
+
+# RUN: %lldb -b -o "disassemble -b -n main" %p/Inputs/riscv_xqci.out | FileCheck %s
+
+# CHECK: 051f 0058 0000  qc.e.li a0, 0x58
+# CHECK: 1516              qc.extu a0, a0, 0x6, 0x0
+
+# RUN: %lldb -b -o "disassemble -b -n main -Y -xqci,+c" %p/Inputs/riscv_xqci.out | FileCheck --check-prefix=CHECK-NOXQCI %s
+
+# CHECK-NOXQCI: <unknown>
```

Comment:
Add CHECK-NOXQCI-NOT to check you don't get some `<unknown>` **and** the proper disassembly here.